src/components/user/populateUserHook.js

```diff
@@ -0,0 +1,9 @@
+const { userRepo } = require('./repo/index');
+
+const populateUser = async (context) => {
```

Comment:
// Write please comment that is should move to application context after it is added.

Reply:
done